projects/bootloader/src/stm32f0xx/jump_to_application.c

```diff
@@ -4,7 +4,10 @@
 #include <stdnoreturn.h>
 #include <string.h>
 
+#include "bootloader_crc32.h"
```

Comment:
Do you need these includes? it doesn't look like you use anything from them in any of these files